### PR TITLE
Test with py3.6 on Travis

### LINKED ISSUE #33: Test with Python 3.6

### PR DESCRIPTION
**Pull request by nicoddemus**
Fix #33

### CODE REVIEW

**Comment by gqmelo** [comment]
Why is it failing for python 3.5?

**Comment by nicoddemus** [comment]
Argh... Travis does not come with `3.5` or `3.6` by default, it is [installed on the fly](https://travis-ci.org/ESSS/conda-devenv/jobs/215628992#L111):

```
3.6 is not installed; attempting download
Downloading archive: https://s3.amazonaws.com/travis-python-archives/binaries/ubuntu/12.04/x86_64/python-3.6.tar.bz2
```

Travis uses the `python` directive to decide which Python to download and install:

```yaml
python:
  - "3.6"
```

Not sure how I can make it work like this... 😞

**Comment by gqmelo** [comment]
How was it working before?

**Comment by nicoddemus** [comment]
Rá: https://github.com/pytest-dev/pytest/blob/master/.travis.yml

**Comment by tadeu** [comment]
Might help also: http://stackoverflow.com/questions/37016688/how-to-run-tox-with-travis-ci

**Comment by nicoddemus** [comment]
Previously we were testing using 3.5. 2.7 and 3.4 seem to be always available.

**Comment by tadeu** [comment]
It seems you can pass multiple versions in the `python:` section of `.travis.yml`

**Comment by nicoddemus** [comment]
You can, but then we will have a product of environments in the `tox` file... but fear not, adopted the same solution that we use on pytest. 😉

**Comment by gqmelo** [comment]
Why not like this: https://github.com/gqmelo/exec-wrappers/blob/master/.travis.yml ?

**Comment by nicoddemus** [comment]
You mean this trick?

```
"TOX_ENV=${TRAVIS_PYTHON_VERSION/[0-9].[0-9]/py${TRAVIS_PYTHON_VERSION/.}}"
```

TBH I prefer my solution. 😃

**Comment by gqmelo** [comment]
Yeah, I meant that, but ok

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,18 @@ language: python
...
   - TOXENV=linting
   - TOXENV=docs
 
 python:
   - "3.5"
+
+matrix:
+  include:
+    - env: TOXENV=py36
+      python: '3.6-dev'
```
Comment (gqmelo) [inline .travis.yml]: why ```3.6-dev```?

Comment (nicoddemus) [inline .travis.yml]: Good question... I copied over from pytest. Let's me try just `3.6`.

**Comment by gqmelo** [comment]
I think it would be nice to also add ```nightly``` to anticipate any problems with future versions.

**Comment by gqmelo** [comment]
You may also be interested in the new travis' cron feature. We could run it at least once a week

**Comment by nicoddemus** [comment]
Of course the AppVeyor failures can be ignored because this branch does not contain an `appveyor.yml` file yet. 😅

**Comment by nicoddemus** [comment]
> You may also be interested in the new travis' cron feature. We could run it at least once a week

Done, I have set it up to run once a week. 👍

**Pull request merged**